src/app/_components/RecentBlocks/StxBlock.tsx

```diff
@@ -0,0 +1,309 @@
+import { Flex, HStack, Icon, Stack, StackProps, StackSeparator } from '@chakra-ui/react';
+import { CaretRight, Circle, CircleDashed, Dot } from '@phosphor-icons/react';
+
+import { Block } from '@stacks/blockchain-api-client';
+
+import { useGlobalContext } from '../../../common/context/useGlobalContext';
+import { buildUrl } from '../../../common/utils/buildUrl';
+import { Link } from '../../../ui/Link';
+import { Text } from '../../../ui/Text';
+import BitcoinCircleIcon from '../../../ui/icons/BitcoinCircleIcon';
+import StxSquareIcon from '../../../ui/icons/StxSquareIcon';
+import { BORDER_WIDTH, BTC_BLOCK_MIN_WIDTH, RING_WIDTH, SMALL_RING_WIDTH } from './consts';
+import { formatTimestamp } from './utils';
+
+export function StxBlockGroup({
+  btcBlockHeight,
+  btcBlockTime,
+  stxBlocks,
+  newestStxBlockHeight,
+}: {
+  btcBlockHeight: number;
+  btcBlockTime: number;
+  stxBlocks: Block[];
+  newestStxBlockHeight: number;
+}) {
+  const network = useGlobalContext().activeNetwork;
+
+  return (
+    <Flex
+      className="stx-block-group"
+      bg={
+        'linear-gradient(to bottom, var(--stacks-colors-redesign-border-primary), var(--stacks-colors-surface-secondary))'
+      }
+      padding={`${BORDER_WIDTH}px`}
+      borderRadius={`calc(var(--stacks-radii-redesign-lg) + ${BORDER_WIDTH}px)`}
+      css={{
+        '&:hover:not(:has(.stx-block:hover)):not(:has(.newest-stx-block:hover))': {
+          background:
+            'linear-gradient(to bottom, var(--stacks-colors-accent-bitcoin-500), var(--stacks-colors-surface-secondary))',
+          '& .btc-block': { bg: 'surfacePrimary' },
+        },
+      }}
+      cursor={'pointer'}
+      align={'stretch'}
+    >
+      <Stack
+        position="relative"
+        gap={2}
+        bg="surfaceSecondary"
+        borderRadius="redesign.lg"
+        p={2}
+        whiteSpace={'nowrap'}
+        justify="space-between"
+      >
+        <HStack
+          gap={1.5}
+          px={1.5}
+          py={1}
+          borderRadius={'redesign.sm'}
+          className="btc-block"
+          w="fit-content"
+          asChild
```

Comment:
What is the asChild doing? If I understand this correctly, all these styles will be applied to the link? Also, there is a gap prop but there is only one child so the gap isnt needed